motr/st/utils/motr_di_corruption_detection_st.sh

```diff
@@ -0,0 +1,139 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
```

Comment:
Replace this line with as this file is added this year
# Copyright (c) 2022 Seagate Technology LLC and/or its Affiliates

Reply:
done